src/runtime/app.config.ts

```diff
@@ -11,7 +11,7 @@ const table = {
     selected: 'bg-gray-50 dark:bg-gray-800/50'
   },
   th: {
-    base: 'text-left',
+    base: 'ltr:text-left rtl:text-right',
```

Comment:
What about when not using either `ltr` or `rtl`? The text is centered for me.

Reply:
When converting the direction to rtl, the writing is not in the middle, so I set the direction for the text
![Screenshot_1](https://github.com/nuxtlabs/ui/assets/123465610/47b1a845-639a-4b73-b091-1772be13d627)

Reply:
When converting the direction to rtl, the writing is not in the middle, so I set the direction for the text
![Screenshot_1](https://github.com/nuxtlabs/ui/assets/66673458/8274e2f8-8bc5-4069-9d36-509cbfffdebe)

Reply:
I mean, we should also let `text-left` as by default `ltr` will only apply if a `dir` attr is present on HTML. See https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support
<img width="771" alt="CleanShot 2023-06-20 at 16 47 39@2x" src="https://github.com/nuxtlabs/ui/assets/739984/318cbcac-da61-4604-90a7-3782850af4eb">

Reply:
Yes, I understood what you meant and made the new commit

Reply:
Shouldn't it be the same for all the `text-left`?

Reply:
Yes , you can Check now